<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="QLabel01">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>96</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;검색어를 입력하세요&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableView" name="csv_sample">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>741</width>
      <height>221</height>
     </rect>
    </property>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>5</number>
    </attribute>
   </widget>
   <widget class="QTextBrowser" name="save_path">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>17</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0
</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;저장할 폴더를 선택하세요&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
   </widget>
   <widget class="QPushButton" name="save_path_find">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>15</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
   <widget class="QLabel" name="QLabel04_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>7</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>저장할 폴더</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="key_word">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>106</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 0
</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="craw_start">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>104</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>실행하기</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pre_anno">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>67</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사전규격</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="real_anno">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>67</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>본공고</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
